local-libs/traceviewer-libs/react-components/style/dialog-component-style.css

```
@@ -14,30 +14,30 @@
     padding: 2px;
 }
 
-.dialog-header{
+.dialog-header {
     background-color: var(--trace-viewer-statusBar-background);
     color: var(--trace-viewer-statusBar-foreground);
     padding: 5px;
```

Stylelint (17.14.0)

[error] Duplicate property "padding" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)